### ISSUE TITLE
crash on nixos

### ISSUE THREAD

**Issue by hans-chrstn** [Duckonaut/split-monitor-workspaces#62]
im running this on a flake-homemanager setup.
i suppose it's because im running an overlay through the latest build?

```
error: builder for '/nix/store/xj179i5x5drhhirj6xgcns9nd90c6km4-split-monitor-workspaces-0.1.drv' failed with exit code 1;
       last 10 log lines:
       > Running phase: buildPhase
       > build flags: -j4
       > [1/2] Compiling C++ object libsplit-monitor-workspaces.so.p/src_main.cpp.o
       > FAILED: libsplit-monitor-workspaces.so.p/src_main.cpp.o
       > g++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I../include -I/nix/store/gmx8cv2sbkhv2m827ldkgl6zl5cw5l8f-hyprland-0.37.1+date=2024-03-29_53aa184-dev/include -I/nix/store/gmx8cv2sbkhv2m827ldkgl6zl5cw5l8f-hyprland-0.37.1+date=2024-03-29_53aa184-dev/include/hyprland/protocols -I/nix/store/gmx8cv2sbkhv2m827ldkgl6zl5cw5l8f-hyprland-0.37.1+date=2024-03-29_53aa184-dev/include/hyprland/wlroots -I/nix/store/aldmf9w7jiss99b93dfndd1nsb9s5qn7-pixman-0.43.4/include/pixman-1 -I/nix/store/bf9ay9lix3w9830i65a4pvwjznx3ggc5-libdrm-2.4.120-dev/include -I/nix/store/bf9ay9lix3w9830i65a4pvwjznx3ggc5-libdrm-2.4.120-dev/include/libdrm -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23
 -DWLR_USE_UNSTABLE -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_main.cpp.o -MF libsplit-monitor-workspaces.so.p/src_main.cpp.o.d -o libsplit-monitor-workspaces.so.p/src_main.cpp.o -c ../src/main.cpp
       > ../src/main.cpp:6:10: fatal error: hyprland/src/helpers/Workspace.hpp: No such file or directory
       >     6 | #include <hyprland/src/helpers/Workspace.hpp>
       >       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > compilation terminated.
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/xj179i5x5drhhirj6xgcns9nd90c6km4-split-monitor-workspaces-0.1.drv'.
error: 1 dependencies of derivation '/nix/store/ag8i7lnasspc6yjd7qqprf7w984smw89-hm_hyprhyprland.conf.drv' failed to build
error: 1 dependencies of derivation '/nix/store/f4k6h36dwwfp0ib5shc5frn01bzhaji4-activation-script.drv' failed to build
error: 1 dependencies of derivation '/nix/store/k2klq770qf00f7dn5il5x2hpk9b4vpkx-home-manager-generation.drv' failed to build
```

**Comment by reDpz** [comment]
Same error here

**Comment by markaacosta** [comment]
same

**Comment by haennes** [comment]
same here

**Comment by n0nvme** [comment]
Same issue on Arch Linux with hyprland v0.38.0. Trying to install plugin using hyprpm

**Comment by elythh** [comment]
Here's how to fix that error https://github.com/Duckonaut/split-monitor-workspaces/pull/59

**Comment by Duckonaut** [comment]
Fixed in #52
